openlibrary/plugins/upstream/account.py

```diff
@@ -348,6 +348,15 @@ def POST(self):
                 ol_account = OpenLibraryAccount.get(email=audit['ia_email'])
                 if ol_account and ol_account.get_user().get_safe_mode() == 'yes':
                     web.setcookie('sfw', 'yes', expires=expires)
+                if (
+                    ol_account
+                    and 'yrg_banner_pref' in ol_account.get_user().preferences()
+                ):
+                    web.setcookie(
+                        ol_account.get_user().preferences()['yrg_banner_pref'],
+                        '1',
+                        expires=(3600 * 24 * 30),
```

Comment:
```suggestion
                        expires=(3600 * 24 * 365),
```
This shouldn't expire for one year.